ebml_matroska.xml

```diff
@@ -1452,7 +1453,7 @@ the last frame it includes, especially for the `ChapterAtom` using the last fram
 When disabled, the movie **SHOULD** skip all the content between the TimeStart and TimeEnd of this chapter; see (#chapter-flags) on Chapter flags.</documentation>
     <extension type="other document" spec="control-track"/>
   </element>
-  <element name="ChapterSegmentUUID" path="\Segment\Chapters\EditionEntry\+ChapterAtom\ChapterSegmentUUID" id="0x6E67" type="binary" range="&gt;0" length="16" maxOccurs="1">
+  <element name="ChapterSegmentUUID" path="\Segment\Chapters\EditionEntry\+ChapterAtom\ChapterSegmentUUID" id="0x6E67" type="binary" length="16" maxOccurs="1">
     <documentation lang="en" purpose="definition">The SegmentUUID of another Segment to play during this chapter.</documentation>
     <documentation lang="en" purpose="usage notes">The value **MUST NOT** be the `SegmentUUID` value of the `Segment` it belongs to.</documentation>
     <implementation_note note_attribute="minOccurs">ChapterSegmentUUID **MUST** be set (minOccurs=1) if ChapterSegmentEditionUID is used; see (#medium-linking) on medium-linking Segments.</implementation_note>
```

Comment:
I miss here the restriction that 1 bit must be 1.

Reply:
> The value **MUST NOT** be the `SegmentUUID` value of the `Segment` it belongs to.

This is a valid `SegmentUUID` elsewhere, so it has exactly the same constraints.

Reply:
I meant to quote that one

> The SegmentUUID of another Segment to play during this chapter

It's better to keep the constraint in fewer locations in case they need to be adjusted.